usaspending_api/accounts/views/common.py

```diff
@@ -0,0 +1,25 @@
+from usaspending_api.awards.serializers import FinancialAccountsByAwardsSerializer
```

Comment:
Poor File C data, we never know where to put it. I figured out that one reason it's hard to find this in the code base is that the model for it lives in the `awards` application and the view for it lives in the `accounts` application. 

If we had to pick one, I'd go with `accounts`, like you have it here (because that data ultimately can only come from an agency DATA Act submission and not from current USAspending info). Just marking your cards that the next time we have a chance, we might want to move the model over to `accounts` so they can live together.

Lastly, since this data is keyed by TAS, what do you think about moving this single view into `tas.py` and getting rid of `common.py`?